### DISCUSSION TITLE
Flicker Fix: Gamescreen pixel-check alternative

### DISCUSSION THREAD

**Discussion by VagabondFragments** [Lailloken/Exile-UI#218] [Ideas & Feedback]
Current design: Gamescreen pixel-check (screen-check) uses the bar above your current location in the top right of the screen, but is only available when map overlay is active.

Update: Give option to pixel-check using the midscreen "Show Mini Life Bar Above You" as this bar takes overlay priority over name-plates that would otherwise cover the current pixel-check. Obviously color variations would have to be established for low hp, ES, and reserved life. This seems like it could fix the current issue of flickering, while also allowing players to use the corner map without losing their clone-frames.

**Comment by Lailloken** [comment]
![image](https://user-images.githubusercontent.com/61888437/230638904-e671a6d6-bc0d-4948-ba80-664f841e1dba.png)

These are two screenshots on top of each other, the second taken after stepping to the side a few steps. The HP bars are not static but move up and down depending on character size and terrain. AHK's pixel-reading becomes really inefficient once you start reading larger areas / longer lines of pixels, which then causes stalling/delay.

That said, it's been a while since I did my testing so I could revisit it in the near future. EDIT: Just not now, right before league-launch.

**Reply by VagabondFragments** [reply]
Thanks for the quick response; I appreciate the effort, especially so close to league-launch. I hope that some resolution can be made in the future, and I'll be keeping a close eye on the project! Thank you for all your work on this as well and hope you have a great league-start.
